esp/bindings/http/platform/httpbinding.cpp

```diff
@@ -2047,7 +2047,7 @@ int EspHttpBinding::onFinishUpload(IEspContext &ctx, CHttpRequest* request, CHtt
     else
     {
         StringBuffer msg;
-        WARNLOG("Exception(s) in EspHttpBinding::onFinishUpload - %s", me->errorMessage(msg).append('\n').str());
+        IWARNLOG("Exception(s) in EspHttpBinding::onFinishUpload - %s", me->errorMessage(msg).append('\n').str());
```

Comment:
OWARNLOG might be better here.

Reply:
O should be used if the error can be traced back to something that the operator needs to configure and fix.  In this case there isn't any reason why the failure is caused by something under their control.

Reply:
@mayx I feel this should remain I for the reasons mentioned by Gavin